extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -206,6 +206,26 @@ $(function () {
 			return result;
 		}
 
+		function trackSave() {
+			Wikia.Tracker.track({
+				action: Wikia.Tracker.ACTIONS.CLICK,
+				category: 'special-curated-content',
+				label: 'save',
+				trackingMethod: 'analytics'
+			});
+		}
+
+		function trackSaveError() {
+			Wikia.Tracker.track({
+				action: Wikia.Tracker.ACTIONS.CLICK,
+				category: 'special-curated-content',
+				label: 'save-error',
+				trackingMethod: 'analytics'
+			});
+		}
```

Comment:
or even you can move label to var and when promise is resolved either with error or success call track function with proper label.